app/src/main/java/de/hhn/aib/labsw/blackmirror/controller/API/Websockets/MirrorApiWebsockets.kt

```diff
@@ -0,0 +1,118 @@
+package de.hhn.aib.labsw.blackmirror.controller.API.Websockets
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import okhttp3.*
+import java.io.IOException
+
+class MirrorApiWebsockets : WebSocketListener(),MirrorApi {
+    var sessions: ArrayList<WebSocket> = ArrayList()
+    private var mapper: ObjectMapper = ObjectMapper()
+    var listeners: HashMap<String, ArrayList<ApiListener>?> =
+        HashMap()
+
+    fun getJSONMapper(): ObjectMapper {
```

Comment:
kotlin uses getter properties syntax, see https://kotlinlang.org/docs/properties.html#getters-and-setters